src/AutoRest.CSharp/Common/AutoRest/Plugins/Configuration.cs

```diff
@@ -150,6 +153,9 @@ public enum UnreferencedTypesHandlingOption
         public static bool SkipSerializationFormatXml { get; private set; }
         public static bool DisablePaginationTopRenaming { get; private set; }
         public static bool GenerateModelFactory { get; private set; }
+
+        public static bool PublicDiscriminatorProperty { get; private set; }
```

Comment:
It would be good to describe the meaning of each property.

Reply:
I added some comments